HTTPServerProject.Tests/RequestTests.cs

```diff
@@ -0,0 +1,50 @@
+namespace HTTPServerProject.RequestTests;
+
+public class UnitTestsForRequests
+{
+    public List<string> request = new List<string>() { "GET / HTTP/1.1", "Host: localhost:5000", "User-Agent: curl/7.79.1", "Accept: */*", "", "quit" };
+
+    [Fact]
+    public void GetRequestInitialLineTest()
+    {
+        var expected = "GET / HTTP/1.1";
+        var reader = new TestReadStreams(request);
+
+        var header = new Header(reader);
+        var initialLine = header.GetLine();
+
+        Assert.Equal(initialLine, expected);
+
+    }
+
+    [Fact]
+    public void GetRequestHeadersTest()
+    {
+        var expected = new List<string>() { "Host: localhost:5000", "User-Agent: curl/7.79.1", "Accept: */*" };
+        var reader = new TestReadStreams(request);
+
+        var header = new Header(reader);
+        var initialLine = header.GetLine();
+        var headers = header.GetHeaders();
+
+        Assert.Equal(headers, expected);
+
+    }
+
+    [Fact]
+    public void GetRequestBodyTest()
+    {
+        var reader = new TestReadStreams(request);
+
+        var header = new Header(reader);
+        var initialLine = header.GetLine();
+        var headers = header.GetHeaders();
+
+        var body = new Body(reader);
```

Comment:
This test looks like it belongs with the body. 

There's a typical pattern of every class having a corresponding test class. It's okay if you're not using that pattern, but if so it needs to be done with intention. I'm not sure if that's there.